### ISSUE TITLE
基本設定討論

### ISSUE THREAD

**Issue by alexhongkonger** [alexhongkonger/hongkongjobsdata#1]
開放討論以下設定:
1. Development Language
2. 初步包括邊幾個求職網
3. 用咩做 Storage
4. Data format
5. Output format

**Comment by alexhongkonger** [comment]
1. Python / Ruby
2. Jobsdb + Indeed + Wanted + jooble + hkgoodjobs + recruit + jobmarket
3. MongoDB
4. JobTitle, Job Desc, Location, Company Name, Reference, First Posted Date
5. Xml + Json

**Comment by ITsLoveCats** [comment]
schedule daily update last 24 hours posts?
it may not possible to parse all data from scratch

**Comment by alexhongkonger** [comment]
yes, it should be a schedule task with limited updated time only.
